scripts/xenadmin-build.sh

```diff
@@ -221,14 +221,17 @@ cp ${REPO}/XenAdmin/bin/Release/{CommandLib.pdb,${BRANDING_BRAND_CONSOLE_NO_SPAC
 
 cd ${OUTPUT_DIR} && zip -r -m  ${BRANDING_BRAND_CONSOLE_NO_SPACE}.Symbols.zip *.pdb
 
-msi_checksum_with_file_name=`sha256sum ./CitrixHypervisorCenter.msi`
+msi_checksum_with_file_name=`sha256sum ./${BRANDING_BRAND_CONSOLE_NO_SPACE}.msi`
 echo $msi_checksum_with_file_name > ${OUTPUT_DIR}/${BRANDING_BRAND_CONSOLE_NO_SPACE}.msi.checksum
 msi_checksum=($msi_checksum_with_file_name)
 
 sha256sum ${OUTPUT_DIR}/${BRANDING_BRAND_CONSOLE_NO_SPACE}-source.zip > ${OUTPUT_DIR}/${BRANDING_BRAND_CONSOLE_NO_SPACE}-source.zip.checksum
 
 echo "INFO: Generating CHCUpdates.xml"
 
+# UPDATE_URL points at the updates XML, we need to point to the MSI
+msi_updates_url="${UPDATES_URL/CHCUpdates.xml/$BRANDING_BRAND_CONSOLE_NO_SPACE.msi}"
```

Comment:
isnt this just msi url? 
we use updates_url elsewhere so could casue confusion

Reply:
✔️